### ISSUE TITLE
Optimize pages - Remove out of view pages from dom

### ISSUE THREAD

**Issue by sameer-shelar** [metafizzy/infinite-scroll#817]
Hi,
Are the pages removed from dom once the user scrolls past a certain number on?
If there are hundred of pages, keeping all the pages in dom will be memory consuming especially on mobile.
Are there any plans to add the feature to remove pages from dom and add a prepend functionality of user scrolls back up?

**Comment by desandro** [comment]
> Are the pages removed from dom once the user scrolls past a certain number on?

No. Infinite Scroll does not manage removing older content from the DOM. Although, you could build this functionality yourself with [Infinite Scroll's events](https://infinite-scroll.com/events.html)

**Add a 👍  reaction** to this issue if you would like to see this feature added. Do not add +1 comments — [They will be deleted](http://metafizzy.co/blog/use-github-reactions-delete-plus-1-comments/).

**Comment by MKaidan** [comment]
Any chance of making this happen?
